wynnset/tapestry-wp
# Delete YoutubeMedia.vue

Delete the wrong YoutubeMedia.vue file.
The correctly capitalized YouTubeMedia.vue is still in master branch

Steps to test:
- Load the PR and make sure it compiles
- Make sure the pulled files have the correctly capitalized YouTubeMedia.vue in the `src/components/lightbox` directory
- Maybe load a YouTube video into a node in your tapestry and check everything displays as expected
